ingestion_server/ingestion_server/indexer.py

```diff
@@ -39,26 +40,28 @@
 """
 
 # For AWS IAM access to Elasticsearch
-AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID", "")
-AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY", "")
-ELASTICSEARCH_URL = os.environ.get("ELASTICSEARCH_URL", "localhost")
-ELASTICSEARCH_PORT = int(os.environ.get("ELASTICSEARCH_PORT", 9200))
-AWS_REGION = os.environ.get("AWS_REGION", "us-east-1")
-
-DATABASE_HOST = os.environ.get("DATABASE_HOST", "localhost")
-DATABASE_USER = os.environ.get("DATABASE_USER", "deploy")
-DATABASE_PASSWORD = os.environ.get("DATABASE_PASSWORD", "deploy")
-DATABASE_NAME = os.environ.get("DATABASE_NAME", "openledger")
-DATABASE_PORT = int(os.environ.get("DATABASE_PORT", 5432))
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID", "")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY", "")
+
+ELASTICSEARCH_URL = os.getenv("ELASTICSEARCH_URL", "localhost")
+ELASTICSEARCH_PORT = int(os.getenv("ELASTICSEARCH_PORT", 9200))
+
+AWS_REGION = os.getenv("AWS_REGION", "us-east-1")
+
+DATABASE_HOST = os.getenv("DATABASE_HOST", "localhost")
+DATABASE_PORT = int(os.getenv("DATABASE_PORT", 5432))
+DATABASE_USER = os.getenv("DATABASE_USER", "deploy")
+DATABASE_PASSWORD = os.getenv("DATABASE_PASSWORD", "deploy")
+DATABASE_NAME = os.getenv("DATABASE_NAME", "openledger")
```

Comment:
TIL there's no difference between `os.getenv` and `os.environ.get` 😮 https://stackoverflow.com/questions/16924471/difference-between-os-getenv-and-os-environ-get

Reply:
Woah, TIL as well!

Reply:
TILx3